webapp/src/js/controllers/inbox.js

```diff
@@ -263,28 +263,24 @@ var feedback = require('../modules/feedback'),
       });
     });
 
-    $transitions.onStart({}, function(trans) {
-      if (trans.to().name.indexOf('reports') === -1 || trans.to().name.indexOf('contacts') === -1 || trans.to().name.indexOf('tasks') === -1 || trans.to().name.indexOf('messages.detail') === -1) {
-        $scope.unsetSelected();
+    $transitions.onBefore({}, (trans) => {
```

Comment:
I switched displaying the popup from `onStart` hook to `onBefore` hook because it makes more sense to not start the transition if we intend to stop it. 
https://ui-router.github.io/guide/transitions#lifecycle-events